packages/core/data-transfer/types/index.d.ts

```diff
@@ -3,4 +3,4 @@ export * from './providers';
 export * from './transfer-engine';
 export * from './utils';
 export * from './encryption';
-export * from './remote';
+export * as remote from './remote';
```

Comment:
Is there a reason that remote and protocol are exported with names but not (for example) encryption?

Reply:
Not really, I can remove that

Reply:
Done :heavy_check_mark: